hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+Improve the type rendered in `st.from_type`, which improves the coverage
```

Comment:
```suggestion
Improve the type rendered in :func:`~hypothesis.strategies.from_type`, which improves the coverage
```
We'd like this to show up as a clickable cross-reference, which requires a bit of extra syntax.

(and while plain backticks try to generate a cross-ref, st.from_type won't be recognized as a link target and thus falls back to italics rather than monospace as you might expect from markdown.  Annoying bit of trivia you just have to memorize!  You can see how this renders at https://hypothesis--3681.org.readthedocs.build/en/3681/changes.html#current-pull-request )

Reply:
Ah, I always forget that .rst works different to .md.... Thanks for the hint!